build.gradle.kts

```diff
@@ -1,21 +1,25 @@
 import org.ajoberstar.grgit.Grgit
 
-// needed for fabric to know where FF executor is....
 buildscript {
     repositories {
-        mavenCentral()
+        maven {
+            name = "EngineHub"
+            url = uri("https://maven.enginehub.org/repo/")
+        }
         maven {
             name = "Fabric"
             url = uri("https://maven.fabricmc.net/")
         }
     }
     dependencies {
-        classpath("net.fabricmc:fabric-loom:${versions.loom}")
+        // needed for fabric to know where FF executor is....
```

Comment:
Is this comment now just referring to the empty dependencies block? Or is it a mistake?